tests/teststun.nim

```diff
@@ -0,0 +1,82 @@
+{.used.}
+
+import options, strutils
+import bearssl
+import ../webrtc/stun/stun_protocol
+import ../webrtc/stun/stun_attributes
+import ../webrtc/stun/stun_utils
+import ./asyncunit
+
+proc newRng(): ref HmacDrbgContext =
```

Comment:
This is how it is done here https://github.com/status-im/nim-eth/blob/master/eth/keys.nim#L66, do you know what's the difference? Moreover, we don't need this for testing, only for production code, as it's probably slower.

Reply:
`TestRng` can just use the regular `rand` Nim proc.